examples/htool_console.c

```diff
@@ -244,6 +244,40 @@ static int set_uart_config(struct libhoth_usb_device *dev,
 
 int htool_console_run(struct libhoth_usb_device *dev,
                       const struct htool_console_opts *opts) {
+  if (opts->snapshot) {
```

Comment:
nit: If snapshot has so little overlap with the existing `htool_console_run` implementation, it may as well be its own function, `htool_console_snapshot`.  (Not necessary to have a different user-visible command, just nitpicking on code structure).

Reply:
Moved to a separate function `htool_console_snapshot`.